### ISSUE TITLE
Testing grobid on Play With Docker

### ISSUE THREAD

**Issue by appukuttan-shailesh** [kermitt2/grobid#696]
I wanted to test out grobid as part of an online tool. I was trying to upload the image on [play-with-docker.com](url) to be able test is easily. I followed the instructions here: https://grobid.readthedocs.io/en/latest/Grobid-docker/

`docker pull lfoppiano/grobid:0.6.1`
and
`docker run -t --rm --init lfoppiano/grobid:0.6.1`

Everything seems to have gone smoothly and I see:
> INFO  [2021-01-19 12:29:45,735] org.eclipse.jetty.server.Server: Started @8445ms

I just can't figure out how to access the service now. I tried this: https://github.com/play-with-docker/play-with-docker#how-can-i-connect-to-a-published-port-from-the-outside-world

But can't seem to get the URL to work. I realize this question might be more to do with 'play with docker', but was hopeful that somebody here could help. Thanks.

**Comment by lfoppiano** [comment]
Dear @appukuttan-shailesh, 
  if you connect to `localhost` does it work? 

In our guide we recommend to map the port to `8080`: 

> docker run -t --rm --init -p 8080:8070 -p 8081:8071 lfoppiano/grobid:${latest_grobid_version}

and then connect to `localhost:8080`. Have you tried that?

**Comment by appukuttan-shailesh** [comment]
Thanks for your reply @lfoppiano 
After seeing your message I did the mapping to port 8080, and then it was very straightforward. play-with-docker (PWD) directly gives you buttons to open the ports at 8080 and 8081. I will attach a screenshot just in case it helps anyone in future trying the same:

![Peek 2021-01-21 09-49](https://user-images.githubusercontent.com/24866517/105326570-2b227080-5bce-11eb-8995-72ead1ea51d5.gif)

Previously, I had imagined that without the mapping, I will simply be able to ask PWD to open port 8070, but that doesn't seem to work and returns a page with "ERR_EMPTY_RESPONSE".
